codestodians/food-basket-front-end
# Deprecate react-bootstrap css framework.

#### What does this PR do?
- Ensures that no component uses the react-bootstrap css component.

#### Description of Task to be completed?
- Refactor the Signin and Signup components to use the material-ui css framework.
- Reduce the width of the Modal component.
- Remove the react-bootstrap dependency.

#### How should this be manually tested?
- Start the application with `yarn start`.
- Click on the signup and signin forms.
- You should see the forms rendered in a material-ui fashion.

#### Any background context you want to provide?
- Material-UI is the new CSS framework used through the project, hence, it becomes necessary to ensure that all components conform to this standard.

#### What are the relevant pivotal tracker stories?
- [153101434](https://www.pivotaltracker.com/story/show/153101434)

#### Extra Note:
- The rendered forms are not quite in the best shape yet. However, that will be done in another story. This story is meant to simply ensure that react-bootstrap is taken out of the project.
